src/main/java/com/marek/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,36 @@
+package com.marek.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Data
```

Comment:
Avoid using `@Data` annotation in the entity classes, it can lead to errors when you will have relations between entities, better to use:
```java
@Entity
@Getter
@Setter
@NoArgsConstructor
```